libc/test/src/time/CMakeLists.txt

```diff
@@ -66,6 +66,24 @@ add_libc_unittest(
     libc.src.time.time_utils
 )
 
+add_libc_unittest(
+  ctime_s_test
+  SUITE
+    libc_time_unittests
+  SRCS
+    ctime_s_test.cpp
+  HDRS
+    TmHelper.h
+    TmMatcher.h
+  CXX_STANDARD
+    20
```

Comment:
Oh, I missed this before.  I didn't spot which feature from C++20 you were using in the unit test.  Can you remove this, or point out which feature you needed this change for?